src/Test/Utilities/Shared/Assert/ConditionalFactAttribute.cs

```diff
@@ -44,8 +44,7 @@ public class HasShiftJisDefaultEncoding : ExecutionCondition
 
     public class IsEnglishLocal : ExecutionCondition
     {
-        public override bool ShouldSkip =>
-                System.Globalization.CultureInfo.CurrentCulture != new System.Globalization.CultureInfo("en-US");
+        public override bool ShouldSkip => System.Globalization.CultureInfo.CurrentCulture.ToString() != "en-US";
```

Comment:
We should probably check InstalledUICulture, basically we want to skip some tests on non-English OS.